waku/v2/protocol/waku_rln_relay/waku_rln_relay_utils.nim

```diff
@@ -386,22 +387,53 @@ when defined(rln) or (not defined(rln) and not defined(rlnzerokit)):
 
     return proofBytes
 
-  proc proofVerify*(rlnInstance: RLN[Bn256], data: openArray[byte],
-    proof: RateLimitProof): bool =
+  proc getMerkleRoot*(rlnInstance: RLN[Bn256]): MerkleNodeResult =
+    # read the Merkle Tree root after insertion
+    var
+      root {.noinit.}: Buffer = Buffer()
+      rootPtr = addr(root)
+      getRootSuccessful = get_root(rlnInstance, rootPtr)
+    if (not getRootSuccessful): return err("could not get the root")
+    if (not (root.len == 32)): return err("wrong output size")
```

Comment:
For easier readability and consistency with nwaku's codebase style, please make these if statements multi-line.

```suggestion
    if not getRootSuccessful: 
      return err("could not get the root")
    if root.len != 32: 
      return err("wrong output size")
```

Reply:
Addressed in [ac3975b](https://github.com/status-im/nwaku/pull/1158/commits/ac3975b05d4ba26addd6b7544c4828147253d4cf)